### ISSUE TITLE
Generate player profile if one doesn't exsist

### ISSUE THREAD

**Issue by VoidsShadow** [yumiris/HXE#17]
Recently, some users' in-game settings and checkpoints were not saving. This includes settings applied via the SPV3 Loader. However, Open Sauce settings would save. This inferred that the game/profile data was unable to save. 
Some users also reported that the `%appdata%\SPV3\` folder was empty after playing the game.

The workaround solution to this issue was to manually create a profile in-game.

**Comment by BinToss** [comment]
Reference: brand new settings folder with a user profile generated by the game executable.
[profiles2.7z.gz](https://github.com/yumiris/HXE/files/4974005/profiles2.7z.gz)

**Comment by BinToss** [comment]
CONTEXT: SPV3.2 Repack. I want to resolve the issue of users attempting to use the Launcher to change settings before they create a profile, resulting in their settings not being saved.
Here are our options:
1. Modify HXE to auto-create a profile if none are detected or if LastProf.txt does not exist, since that might be easier.
2. ~~Package and deploy a pre-built UserData folder to the mod's/game's root directory. Currently, the Launcher is hardcoded to use `%appdata%\SPV3\` as the UserData directory so this will not work without modifying the Launcher.~~
3. ~~Don't change the Launcher's `-path` argument. Instead, deploy a pre-built UserData folder to `%appdata%\SPV3\`. Can HXE already do this or does this need to be implemented?~~


EDIT:
Changed "Settings Folder" and "Path" to "UserData directory".

Note: I'll also begin referring to SPV3.exe as the "Launcher" and HXE as the "Loader" to disambiguate them in layman's discussions.

**Comment by BinToss** [comment]
Refer to https://github.com/VoidsShadow/HXE/tree/autoprofile for progress.
